src/components/navbar/navDropdown/NavDropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../utils/variables';
+
+.dropdownContainer {
+    z-index: 1;
+    background-color: $white;
+    position: absolute;
+    display: inherit;
+    top: 0;
+    left: calc(5vw - 10px);
```

Comment:
Is it possible to have this as a static `px` value instead? The position is `absolute` already so we shouldn't need to do viewport calculations.

Reply:
well, the vp calculations scale and pixels don't. This will have to be a question for the design team perhaps. If they want the dropdown to be that picture size only or if they were going for the space between the edges.

Reply:
I'm with Travis here. I may just be a bad person but I love a bit of vw/h as it seems like an easy way to get some responsiveness